static/assets/templates/todo.json

```diff
@@ -2,10 +2,10 @@
     "name": "todo",
     "description": "A simple Todo application with a back-end implementation in every language, and an interactive tutorial for the policy language. Provides a head start for learning how to integrate Aserto into your own app.",
     "assets": {
-        "manifest": "citadel/manifest.yaml",
+        "manifest": "todo/manifest.yaml",
         "policy": {
             "name": "todo",
-            "resource": "ghcr.io/aserto-policies/policy-todo:latest"
+            "resource": "ghcr.io/aserto-policies/policy-todo-rebac:3.0.0-prerelease"
```

Comment:
needs to update

Reply:
This PR will not be merged. It's only here in order to create a preview that can be used in eng.
The actual assets are merged via CI from aserto-dev/topaz.